src/app/views/user/components/account/__tests__/user-bookmarks.spec.js

```diff
@@ -1,7 +1,7 @@
 // import UserBookmarks from '../user-bookmarks.vue'
```

Comment:
same here.

Reply:
I made these changes with grep/sed, hence the changes in unused/commented files